designs/containers/workload-discovery/workload-discovery.md

```diff
@@ -321,8 +321,17 @@ All the tables and relationships indicated on the previous entities will also be
 Out of scope.
 
 ### OSC Synchronization Tasks
-**TBD on next revision**  
-Describe any changes on the OSC internal synchronization tasks or metatasks. Use a diagram to represent any updated or new task graph.
+The security group(SG) membership will be synchronized by OSC using its tasks and metatasks. This synchronization will continue to be triggered, as today, by: security group creation, update or deletion; and notifications received from Kubernetes relevant for the security group.
+The diagram below depicts the tasks and metatasks involved on this change:
+![](./images/discovery-tasks.png)  

```

Comment:
the yes and no should be switched. and port group check meta task needs to run after the label check meta tasks

We can use the job graph convention below to indicate dependencies.

![image](https://user-images.githubusercontent.com/7297623/29189457-7329c6b0-7dcb-11e7-89fd-8226700483c2.png)